scripts/deploy.py

```diff
@@ -1,59 +1,38 @@
 import sys
-import time
-from brownie import (
-    BalancerRewardsController,
-    RewardsManager
-)
+from brownie import RewardsManager
```

Comment:
I'm not sure, do we need a test for deploy script?

Reply:
Usually, I do test for deploy script, but this one contains only contracts deploy call, so I've used contact deploy directly in test as it'll be used on deployment